lib/hipe/icode/hipe_icode_primops.erl

```diff
@@ -735,6 +736,8 @@ type(Primop, Args) ->
       erl_types:t_any();
     build_stacktrace ->
       erl_types:t_list();
+    raw_raise ->
+      erl_types:t_atom();
```

Comment:
Does `raw_race` ever return at all? If not, it's type should be `t_none()`.

Reply:
> Does raw_race ever return at all? If not, it's type should be t_none(). The same applies to the next line too.

@margnus1 Yes, it will return `badarg` if `Class` is not one of `error`, `exit`, or `throw`.